cpp/include/cudf/datetime.hpp

```diff
@@ -207,5 +207,89 @@ std::unique_ptr<cudf::column> is_leap_year(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /** @} */  // end of group
+
+/**
+ * @brief  Round up to the nearest day
+ *
+ * @param[in] cudf::column_view of the input datetime values
+ *
+ * @returns cudf::column of the same datetime resolution as the input column
+ * @throw cudf::logic_error if input column datatype is not TIMESTAMP
+ */
+std::unique_ptr<cudf::column> ceil_day(
+  cudf::column_view const& column,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
+/**
+ * @brief  Round up to the nearest hour
```

Comment:
No extra space after tag.
```suggestion
 * @brief Round up to the nearest hour
```